assets/src/design-system/components/slider/index.js

```diff
@@ -41,6 +41,16 @@ const rangeThumbHover = css`
   background-color: ${({ theme }) => theme.colors.interactiveBg.primaryHover};
 `;
 
+const rangeThumbFocus = css`
+  border: 2px solid ${({ theme }) => theme.colors.accent.secondary};
+  padding: 2px;
+  background-clip: content-box;
+  width: ${({ thumbSize }) => thumbSize + 8}px;
+  height: ${({ thumbSize }) => thumbSize + 8}px;
+  margin-left: ${({ thumbSize, width, percentage }) =>
+    getFocusedThumbMargin(percentage, thumbSize, width)}px;
+`;
```

Comment:
This needs a `z-index: 1`, otherwise the focus ring is behind the lower bar line:

![Screen Shot 2021-02-23 at 12 15 59](https://user-images.githubusercontent.com/637548/108881283-21c36400-75d1-11eb-979b-dacb6457c337.png)

Reply:
Hah! Good catch, my brain was just filling in the missing part so I could see it even though it didn't exist 😄